common4j/src/test/com/microsoft/identity/common/java/crypto/MockData.java

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) Microsoft Corporation.
+// All rights reserved.
+//
+// This code is licensed under the MIT License.
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files(the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and / or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions :
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+package com.microsoft.identity.common.java.crypto;
+
+import static com.microsoft.identity.common.java.AuthenticationConstants.ENCODING_UTF8;
+
+public class MockData {
+    public MockData(){}
+
+    // Value extracted from the legacy StorageHelper.
+    // Data Set 1 - Predefined key.
+    static final byte[] TEXT_TO_BE_ENCRYPTED_WITH_PREDEFINED_KEY = "TEST_TEXT_TO_ENCRYPT".getBytes(ENCODING_UTF8);
```

Comment:
nit: blank lines between these would make it easier to read.

Reply:
I think it'll make it harder to read, as in, harder to group.